README.md

```diff
@@ -53,6 +53,23 @@ pip install retworkx
 will build retworkx for your local system from the source package and install
 it just as it would if there was a prebuilt binary available.
 
+### Optional dependencies
+
+If you're planning to use the `retworkx.visualization` module you will need to
+install optional dependencies to use the functions. The matplotlib drawer
+required that the [matplotlib](https://matplotlib.org/) library is installed.
+This can be installed with `pip install matplotlib` or when you're installing
```

Comment:
required -> requires?

Also, matplotlib requires itself?  Perhaps you're trying to say that retworkx.visualization requires that matplotlib is installed for [drawing|rendering] functionality?

Reply:
It's a bit confusing the way I wrote it here without prior familiarity. In this upcoming release there are 2 visualization functions `mpl_draw` (which is written using matplotlib) and `pydot_draw` (which is written using graphviz via pydot). The matplotlib drawer is `mpl_draw` here, I've update this to try and make it clearer in:  https://github.com/Qiskit/retworkx/pull/339/commits/500e8923227d787ee51dd1e65c71a48e737dd77d